data/transactions/logic/eval.go

```diff
@@ -180,6 +181,7 @@ func Eval(program []byte, params EvalParams) (pass bool, err error) {
 					errstr += sb.String()
 				}
 			}
+			logging.Base().Errorf("recovered panic in Eval: %s", errstr)
```

Comment:
you should include the panic value x in these. see `func (pe PanicError) Error() string`

Reply:
thanks, I'm just `Errorf`ing the `PanicError` now